app/uk/gov/hmrc/agentsexternalstubs/models/LegacyRelationshipRecord.scala

```diff
@@ -32,7 +32,7 @@ case class LegacyRelationshipRecord(
       nino.map(LegacyRelationshipRecord.ninoKey),
       utr.map(LegacyRelationshipRecord.utrKey),
       Option(LegacyRelationshipRecord.agentIdKey(agentId)),
-      utr.map(LegacyRelationshipRecord.agentIdAndUtrKey(agentId, _))
+      utr.map(LegacyRelationshipRecord.agentIdAndUtrKey(agentId, _)),
```

Comment:
do we need this? legacy?

Reply:
this was already there before I checked with master